kubernetes/playgrpc.yml

```diff
@@ -0,0 +1,143 @@
+---
```

Comment:
can we call this file `play-grpc.yml`?

Reply:
Most of the properties on this file are irrelevant, I guess. We should trim it down to the minimum necessary.

I think all RP prefixed vars can be removed for instance.